tokio/src/macros/join.rs

```diff
@@ -96,25 +143,21 @@ doc! {macro_rules! join {
         // <https://internals.rust-lang.org/t/surprising-soundness-trouble-around-pollfn/17484>
         let mut futures = &mut futures;
 
-        // Each time the future created by poll_fn is polled, a different future will be polled first
+        const COUNT: u32 = $($total)*;
+
+        // Each time the future created by poll_fn is polled,
+        // if not running in biased mode,
+        // a different future will be polled first
         // to ensure every future passed to join! gets a chance to make progress even if
         // one of the futures consumes the whole budget.
```

Comment:
Please reflow this text so that the line breaks are at 80 chars. (or whatever line length the rest of this file uses)